Cargo.lock

```diff
@@ -6331,22 +6353,24 @@ dependencies = [
 [[package]]
 name = "webrender"
 version = "0.61.0"
-source = "git+https://github.com/servo/webrender#4ba17a61580a03b68881ffba7ee22bcf8e1aac72"
+source = "git+https://github.com/servo/webrender#5a5cb7951886977254164b967008177aadc275d0"
```

Comment:
Are you deliberately using an older git version? It would be nice to have the latest release to get the dwrote update.